bears/ruby/RuboCopBear.py

```diff
@@ -174,84 +175,87 @@ class members.
         :param allow_unused_method_keyword_args:
             Allows unused keyword arguments in a method.
         """
-        naming_convention = {'camel': 'camelCase', 'snake': 'snake_case'}
```

Comment:
None of this code needs to be modified.

You can add two lines at the beginning:

```
        if rubocop_config:
            return None
```

Then the rest of the code can work as before.